### PR TITLE
Udp sample loss feature

### PR DESCRIPTION
**Pull request by elianalf**
Sample loss mode and example

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/135/)

File: docs/examples/sample_loss.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.. _sample_loss:
+
+Udp Sample Loss modality
+================================
```
Comment (JesusPoderoso) [inline docs/examples/sample_loss.rst:4]: Fit title undeline
```suggestion
Udp Sample Loss modality
========================
```

**Review by JesusPoderoso — CHANGES_REQUESTED** [review]

**Review by JesusPoderoso — APPROVED** [review]
LGTM with green CI

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/136/)

**Review by JesusPoderoso — CHANGES_REQUESTED** [review]
Please, update also all the sceenshots to display both "Active Statistics" & "Active Monitor Service" as checked

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/138/)

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Pull request merged**